internal/nostr/nostr.go

```diff
@@ -128,31 +128,32 @@ func NewService(ctx context.Context) (*Service, error) {
 
 	var openSubscriptions []Subscription
 	if err := svc.db.Where("open = ?", true).Find(&openSubscriptions).Error; err != nil {
-		logger.Errorf("Failed to query open subscriptions: %v", err)
+		logger.Errorf("failed to query open subscriptions: %v", err)
```

Comment:
can you use JSON logging? logger.WithError(err).Error("failed to query open subscriptions")

Reply:
always, always use JSON logging.

Reply:
Done!